### DISCUSSION TITLE
FrSky Twin Lite included in the current 4 in 1 module

### DISCUSSION THREAD

**Discussion by Crunch69** [pascallanger/DIY-Multiprotocol-TX-Module#980] [Ideas]
Can we make the module a 5 in 1 and include FrSkys Twin Mode!  we would not have to swap moule back and forth and it would make the radio truly complete

**Comment by Quaxbruch** [comment]
Hi is there any news on this? is this doable? I would really like to use the TWGR6 on my Tandem 18s using the module.
